multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -1584,10 +1643,19 @@ def evaluate_coco(
         ----------
         anno_file
             The annotation file in COCO format
+        return_pred
+            Whether to return the prediction result of each row.
+        eval_tool
+            The eval_tool for object detection. Could be "pycocotools" or "torchmetrics".
         """
+        # TODO: refactor this into evaluate()
         if isinstance(anno_file_or_df, str):
             anno_file = anno_file_or_df
-            data = from_coco(anno_file)
+            data = from_coco_or_voc(
```

Comment:
What does this name `from_coco_or_voc` suggest? We support coco and voc format for now? I think we only support coco.

Reply:
Yes we planned to support COCO only, but for faster benchmarking I also add the support for VOC format. Running the benchmark in VOC format was painful if we don't have VOC support and use voc2coco script instead. Later I'll modify the voc2coco.py script to make it easier.